app/assets/javascripts/controllers/middleware_providers/module_controllers.js

```diff
@@ -0,0 +1,144 @@
+miqHttpInject(angular.module('mwProviders.controllers', ['miQStaticAssets']))
+.controller('miqNewProviderController', ['$location', '$http', function($location, $http) {
+  this.onBackToListClick = _.bind(function() {
+    $location.path('/ems_middleware/show_list');
+  }, this);
+
+  this.validateAction = _.bind(function(validateData) {
+    $http.post('/ems_middleware/validate_provider', this.newProvider).then(function (response) {
+      console.log(response.data);
+    });
+  }, this);
+
+  this.formActions = [{
+    btnClass: 'btn-primary',
+    title: 'Add this Middleware Manager',
```

Comment:
these should be `title: __('Add this Middleware Manager'),` (double underscore for i18n)